packages/suite/src/reducers/wallet/coinjoinReducer.ts

```diff
@@ -14,20 +17,42 @@ type CoinjoinState = {
     clients: PartialRecord<Account['symbol'], Client>;
 };
 
+export type CoinjoinRootState = {
+    wallet: {
+        coinjoin: CoinjoinState;
+    };
+};
+
 const initialState: CoinjoinState = {
     accounts: [],
     clients: {},
 };
 
-const createAccount = (draft: CoinjoinState, account: Account) => {
+const createAccount = (
+    draft: CoinjoinState,
+    { account, targetAnonymity }: Extract<Action, { type: typeof COINJOIN.ACCOUNT_CREATE }>,
```

Comment:
I don't like these extract, we are going to get rid of these union and this will only complicate migration. Use same type as for action payload or `ReturnType<typeof actionCreator>` which should work fine also in RTK.